package.json

```diff
@@ -39,18 +39,24 @@
   },
   "dependencies": {
     "@blockworks-foundation/mango-client": "3.4.7",
-    "@project-serum/anchor": "0.24.2",
-    "@solana/spl-token": "0.1.8",
-    "@solana/web3.js": "1.42.0",
     "camelcase": "5.3.1"
   },
+  "peerDependencies": {
```

Comment:
I don't want to force external consumers to use peer dependencies, I would rather keep those as `dependencies` and allow consumers to override through their package manager, or bring our own versions for safety.

Reply:
hm I'm surprised, I thought peerDependencies were useful to avoid packages duplications?

the `uxd_app_frontend` use `uxd-staking` and `uxd-client`, if they both use `"@project-serum/anchor": "0.24.2"`, then the `uxd_app_frontend` will have only one `@project-serum/anchor` package right?

Reply:
Yes, peerDependencies can help to prevent duplication.

But we don't want to use peerDependencies for uxd-client, because we don't want 3rd party users to think about it, they just install the client and it works, and then if they want/need to go further for de-duplication, they can override with their package manager.

Trust me let's just use `dependencies` here, and keep doing what we do with `peerDependencies` in other private packages.